lib/tasks/taxonomy/create_worldwide_taxons.rake

```diff
@@ -0,0 +1,368 @@
+namespace :taxonomy do
+  desc "Create draft taxons for worldwide taxonomy"
+  task create_draft_taxons_for_worldwide_taxonomy: :environment do
+    def remote_taxons
+      @remote_taxons ||= RemoteTaxons.new
+    end
+
+    def taxon_search(country_slug)
+      remote_taxons.search(
+        query: country_slug,
+        states: ["draft"],
+      )
+    end
+
+    def slug(_county, taxon)
+      if taxon[:slug].ends_with?("COUNTRY")
+        taxon[:slug].gsub("COUNTRY", country[:slug])
+      else
+        "#{taxon[:slug]}-#{country[:slug]}"
+      end
+    end
+
+    def internal_name(country, taxon)
+      "#{taxon[:title]} (#{country[:name]})".gsub("COUNTRY", country[:name])
+    end
+
+    def title(country, taxon)
+      (taxon[:title]).to_s.gsub("COUNTRY", country[:name])
+    end
+
+    def country_parent_taxon(country)
+      # TODO: get ALL taxon from publishing api (of type /world)
+
+      taxons = taxon_search(country[:slug])
+      taxons.first.content_id
+    end
+
+    def create_taxon(country, taxon)
+      {
+        path_prefix: "/world",
+        path_slug: slug(country, taxon),
+        internal_name: internal_name(country, taxon),
+        title: title(country, taxon),
+        description: taxon[:description],
+        visible_to_departmental_editors: true, # find out what this is used for
```

Comment:
This determines whether or not the taxon is visible in Whitehall when it's in draft state. `true` is fine here.